$OpenBSD: patch-src_bin_e_main_c,v 1.1 2007/12/04 17:43:15 bernd Exp $

The OpenBSD tar(1) doesn't have a -k option to preserve existing files.
Use pax(1) with its -k option.

--- src/bin/e_main.c.orig	Mon Dec  3 13:59:16 2007
+++ src/bin/e_main.c	Mon Dec  3 13:59:48 2007
@@ -1088,13 +1088,13 @@ _e_main_dirs_init(void)
      {
 	snprintf(buf, sizeof(buf), 
 		 "gzip -d -c < %s/data/other/desktop_files.tar.gz | "
-		 "(cd %s/applications/ ; tar -xkf -)", 
+		 "(cd %s/applications/ ; /bin/pax -rk)", 
 		 e_prefix_data_get(),
 		 efreet_data_home_get());
 	system(buf);
 	snprintf(buf, sizeof(buf), 
 		 "gzip -d -c < %s/data/other/desktop_order.tar.gz | "
-		 "(cd %s/.e/e/ ; tar -xkf -)", 
+		 "(cd %s/.e/e/ ; /bin/pax -rk)", 
 		 e_prefix_data_get(),
 		 homedir);
 	system(buf);
@@ -1105,7 +1105,7 @@ _e_main_dirs_init(void)
      {
 	snprintf(buf, sizeof(buf), 
 		 "gzip -d -c < %s/data/other/efm_favorites.tar.gz | "
-		 "(cd %s/.e/e/ ; tar -xkf -)", 
+		 "(cd %s/.e/e/ ; /bin/pax -rk)", 
 		 e_prefix_data_get(),
 		 homedir);
 	system(buf);
